driver/config/config_test.go

```diff
@@ -47,6 +47,7 @@ func TestViperProvider(t *testing.T) {
 			assert.Equal(t, []string{
 				"http://return-to-1-test.ory.sh/",
 				"http://return-to-2-test.ory.sh/",
+				"http://*.wildcards.ory.sh",
```

Comment:
I would add some more complex use cases here (including those which are being explicitly ignored).

Reply:
This is the only one that went through the filtering. The full list is in internal/.kratos.yaml